File: JAVA PROGRAMMING/Test57.java
// 57. PRINT THE 0 TO N IN THE FORM OF (A^p)
import java.util.Scanner;
public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Range Value: ");
        int range = in.nextInt();
        System.out.println("Enter Power Value: ");
        int power = in.nextInt();
        for(int i = 0; i <= range; i++) {
            System.out.print((int)(Math.pow(i, power))+" ");
        }
        in.close();

    }
}

File: TYPESCRIPT PROGRAMMING/Test57.ts
// 57. PRINT THE 0 TO N IN THE FORM OF (A^p)
const SeriesPower = (range: number, power: number): void => {
    for(let i: number = 0; i <= range; i++) {
        process.stdout.write(`${Math.pow(i, power)} `);
    }
    console.log();
}
SeriesPower(10, 3);
SeriesPower(20, 4);